web_src/js/components/ActivityHeatmap.vue

```diff
@@ -1,8 +1,7 @@
 <template>
   <div id="user-heatmap">
-    <div class="total-contributions">
-      {{ sum }} contributions in the last 12 months
-    </div>
+    <!-- eslint-disable-next-line vue/no-v-html (safely generated in the backend) -->
+    <div class="total-contributions" v-html="locale.contributions_in_the_last_12_months_html"/>
```

Comment:
Is there actual HTML content here or is it just a text string, in which cas `v-html` can be avoided?

Reply:
It has `<gitea-locale-number>`

Reply:
We can avoid it with https://github.com/go-gitea/gitea/pull/24134

Reply:
ok